packages/patrol_cli/lib/src/ios/ios_deploy.dart

```diff
@@ -60,14 +60,20 @@ class IOSDeploy {
 
     var totalTimeElapsed = Duration.zero;
     while (!launchSucceeded) {
+      if (_disposeScope.disposed) {
+        _logger.detail(
```

Comment:
Shouldn't you break here to leave the infinite loop? Or it will not crash but still run.

Reply:
I definitely should! Thanks!